README.md

```diff
@@ -320,11 +326,11 @@ Now add the following content to your newly created `blog_page.html` file:
 Note the use of Wagtail's built-in `get_parent()` method to obtain the
 URL of the blog this post is a part of.
 
-From <http://127.0.0.1:8000/blog/>, click "Add a child page" again, and this time create a new "Blog Page".
+From <http://127.0.0.1:8000/blog/>, click "Add a child page" again, and this will automatically create a new blog page for you.
```

Comment:
The page type will automatically be selected, but it won't automatically be _created_, so maybe we could change the wording a bit?

```suggestion
From <http://127.0.0.1:8000/blog/>, click "Add a child page" again, and this time the "Blog Page" type will automatically be selected.
```